test/gas/gas.js

```diff
@@ -249,7 +288,11 @@ describe("Gas", function() {
             const options = { from, gas: 5000000 };
 
             // prime storage by making sure it is set to 0
-            await instance.methods.reset().send(options);
+            try {
```

Comment:
why do we need to `try`/`catch` here?

Reply:
Sorry! We don't, that code is vestigial from when I was testing.  Removing now :+1: